### DISCUSSION TITLE
Adaptive height for datagrid

### DISCUSSION THREAD

**Discussion by AllForNothing** [vmware/clarity#6352] [Ideas] [answered]
Can you provide an attribute that allows the height of the datagrid to match the number of rows.
It’s common that the rows on the last page are less than the page size

![image](https://user-images.githubusercontent.com/30999793/134283155-66996bd7-a06a-4a97-91dc-0fef4958259b.png)

**Comment by hippee-lee** [comment, marked as answer]
Out of the box, the datagrid does ship with auto height, see a basic case demo [here](https://stackblitz.com/edit/clarity-light-theme-v5-fjjnfb?file=src/app/app.component.html). What I see in your demo is a datagrid with pagination. On the last page of a paginated datagrid we chose to keep the datagrid height the same to prevent a `jumping` effect in the UI when the last page does not have the full number of rows / page as the previous pages do. 

We originally shipped with the way you asked but user feedback preferred the current behavior.
